sqlalchemy_history/unit_of_work.py

```diff
@@ -21,6 +21,12 @@ def __init__(self, manager):
         self.manager = manager
         self.reset()
 
+    def create_version_session(self, session):
+        try:
```

Comment:
Test seems to be working with rebased code, 
can you add small note saying what happened here so if someone comes cold to this code and has no gitlens is not confused.